test/Waives.Http.Tests/Waives.Http.Tests.csproj

```diff
@@ -33,4 +33,8 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This block isn't necessary: it's including an empty folder in the project file, so I think it can be removed.